assets/css/style.scss

```diff
@@ -407,6 +407,23 @@ h2.gala-heading {
     color: $white;
 }
 
+.sponsorship-image {
+  background-image:url(https://res.cloudinary.com/zaigezaigu/image/upload/v1670371900/zgzg-io-website/%E8%B5%9E%E5%8A%A9/jason-goodman-wVh5grSMYaY-unsplash_bupoj8.jpg);
+  background-size: cover;
+  background-repeat:no-repeat;
+  margin: 0 auto 0 auto;
+  min-width: 100%;
+  padding-top:50px;
+}
+
+.sponsor-body {
+  width: 1116px;
```

Comment:
When the browser is narrow, content is cut off:
<img width="547" alt="image" src="https://user-images.githubusercontent.com/594058/206870392-3fe49ffe-3c25-4670-a705-5d39821e29bd.png">